test/Interop/Cxx/class/Inputs/type-classification.h

```diff
@@ -189,4 +189,11 @@ struct StructWithCopyConstructorAndSubobjectCopyConstructorAndValue {
       : member(other.member) {}
 };
 
+struct DummyStruct {};
+
+struct HasUserProvidedDestructorAndDummy {
+  DummyStruct dummy;
+  ~HasUserProvidedDestructorAndDummy() {}
+};
```

Comment:
If this type is interesting for type classification purposes, please update the tests `./test/Interop/Cxx/class/type-classification-*`.